src/types/oec/okex.ts

```diff
@@ -0,0 +1,294 @@
+import { CryptoNodeData, VersionDockerImage } from '../../interfaces/crypto-node';
+import { defaultDockerNetwork, NetworkType, NodeClient, NodeType, Role, Status } from '../../constants';
+import { Docker } from '../../util/docker';
+import { ChildProcess } from 'child_process';
+import { v4 as uuid } from 'uuid';
+import request from 'superagent';
+import fs from 'fs-extra';
+import path from 'path';
+import os from 'os';
+import { Ethereum } from "../ethereum/ethereum"
+import { filterVersionsByNetworkType } from '../../util';
+
+const coreConfig = `
+moniker = "{{MONIKER}}"
+chainid = "exchain-66"
+db_backend = "rocksdb"
+fast_sync = true
+auto_fast_sync = true
+log_level = "info"
+log_format = "plain"
+minimum-gas-prices = "0.0000000001okt"
+
+[rest]
+laddr = "tcp://0.0.0.0:{{RPC_PORT}}"
+
+[p2p]
+laddr = "tcp://0.0.0.0:{{PEER_PORT}}"
+`;
+
+export class OKEX extends Ethereum {
+
+  static versions(client: string, networkType: string): VersionDockerImage[] {
+    client = client || OKEX.clients[0];
+    let versions: VersionDockerImage[];
+    switch (client) {
+      case NodeClient.CORE:
+        versions = [
+          {
+            version: '1.1.4.3',
+            clientVersion: '1.1.4.3',
+            image: 'okexchain/fullnode-mainnet:v1.1.4.3',
+            dataDir: '/root/.exchaind/data',
+            walletDir: '/root/keystore',
+            configDir: '/root/.exchaind/config',
+            networks: [NetworkType.MAINNET],
+            breaking: false,
+            generateRuntimeArgs(data: CryptoNodeData): string {
+              return ' start'
```

Comment:
Is it possible to explicitly pass in the config file as a flag (e.g. `--config /root/config.toml`) rather than implicitly trusting the docker image maintainers to not change the file system configuration?

Reply:
exchaind doesn't have an option for config but it does have one for  `--home` so I've added that in the last commit